create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -382,14 +385,43 @@ void shouldEmitMessageToEventReferenceContainingS3UriPointingToNewCreatePublicat
         assertThat(request, is(equalTo(expectedRequest)));
     }
 
+    @Test
+    void shouldExtractJournalArticle() throws IOException {
```

Comment:
shouldExtractJournalArticleWhenScopusCitationTypeIsArticle

I like to think that test names is the actual code. One should be able to undestand what the converter does just by looking at the test names.

Reply:
I agree, that's a slip

Fixed